tests/test_servo.py

```diff
@@ -130,6 +131,46 @@ def test_save_configuration(connect_to_slave):
     _clean(filename)
 
 
+@pytest.mark.canopen
+@pytest.mark.ethernet
+def test_check_configuration(connect_to_slave, read_config, pytestconfig):
+    servo, net = connect_to_slave
+
+    assert servo is not None and net is not None
+
+    protocol = pytestconfig.getoption("--protocol")
+
+    filename = read_config[protocol]["load_config_file"]
+
+    assert os.path.isfile(filename)
+
+    check_failed_message = re.escape("Configuration check failed for the following registers:")
+
+    # Configuration was not loaded yet, we expect the servo state to be different.
+    with pytest.raises(ILConfigurationError, match=check_failed_message + r".*"):
+        servo.check_configuration(filename)
+
+    # Load the configuration, the subsequent check should no longer raise an error.
+    servo.load_configuration(filename)
+    servo.check_configuration(filename)
+
+    # Change a random register
+    register = "DRV_PROT_USER_OVER_VOLT"
+    new_value = 10.0
+    servo.write(register, data=new_value, subnode=1)
```

Comment:
I think you should reload the configuration after finishing the test or at least restore this register. This would produce a fault when doing a motor enable.

Reply:
Is this resolved? 
shouldn't this be done in a test teardown?